plugins/discovery-gce/src/main/java/org/elasticsearch/discovery/gce/GceUnicastHostsProvider.java

```diff
@@ -78,24 +75,29 @@ public GceUnicastHostsProvider(Settings settings, GceComputeService gceComputeSe
         this.networkService = networkService;
         this.version = version;
 
-        this.refreshInterval = settings.getAsTime(Fields.REFRESH, TimeValue.timeValueSeconds(0));
-        this.project = settings.get(Fields.PROJECT);
-        this.zones = settings.getAsArray(Fields.ZONE);
+        this.refreshInterval = GceComputeService.REFRESH_SETTING.get(settings);
+        this.project = GceComputeService.PROJECT_SETTING.get(settings);
+        this.zones = GceComputeService.ZONE_SETTING.get(settings);
 
-        this.tags = settings.getAsArray(Fields.TAGS);
+        this.tags = GceDiscovery.TAGS_SETTING.get(settings);
         if (logger.isDebugEnabled()) {
-            logger.debug("using tags {}", Arrays.asList(this.tags));
+            logger.debug("using tags {}", this.tags);
```

Comment:
I'm unsure because IIRC, it will print in that case `["a", "b"]` but with your proposal it would print  `[["a", "b"]]`. But may be logger changed since I wrote the original code?